packages/minimongo/modify.js

```diff
@@ -185,20 +185,20 @@ var MODIFIERS = {
   $currentDate: function (target, field, arg) {
     if (typeof arg === "object" && arg.hasOwnProperty("$type")) {
        if (arg.$type !== "date") {
-         throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers");
+         throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers: " + field);
```

Comment:
@s7dhansh Making the more substantial change of confirming to the `Meteor.Error` format is probably worth discussing in an issue if you think it's a problem that it needs addressing.  It may be, but at first glance I'm really not sure if there is a particular reason for why there is a difference.

However, if we changed the `MinimongoError` constructor to take an optional second parameter as @benjamn suggested (perhaps in as an `options` object if you think there is room for expansion), another pull-request after this could easily implement your suggestion by making a change in only one place instead of 29, since all 29 of these lines will need to be changed again. 😉